content/python/get_virtual_guest_usage_records.md

```diff
@@ -11,24 +11,41 @@ tags:
     - "metric"
     - "virtualguest"
 ---
+
+This example will print out the average memory and CPU usage for a given time period for all virtual guests on your account. 
+
+### Time Format
+Use the following time format for any requests. `YYYY-MM-DDTHH:MM:SS` or Epoch Time will work. These times are all tracked in CST, and the response will be in your users local timezone. When requesting data on day boundaries, your start time should be `00:00` and end time should be `23:59`
+
+### Notes
+
+The metric tracking api can be a bit slow, depending on how much data you are requesting, so if your request times out, try lowering how much time you are asking for.
+
+Not all virtual guests will have a metric tracking object, for example any suspended virtual guest, or VMWare guest will not be tracked. 
+ 
+
 ```py
 import SoftLayer
+import sys
+from pprint import pprint as pp 
```

Comment:
I think this import could just be removed. looks like an unused import statement

Reply:
Thanks, removed.